README.md

```diff
@@ -201,15 +207,15 @@ After the deployment is complete, you can try the deployed application by naviga
 
 Most of the Azure resources deployed in the prior steps will incur ongoing charges unless removed. Also a few of the resources deployed go into a soft delete status. It's best to purge those once you're done exploring, Key Vault is given as an example here. Azure OpenAI and Azure Machine Learning Workspaces are others that should be purged.
 
+> **_NOTE:_** To purge the AI services resources, `Microsoft.CognitiveServices/locations/resourceGroups/deletedAccounts/delete` permission must be given to the user on the subscription scope where the resources are deployed. A role assignment at the resource or resource group scopes will be insufficient to access the purge functionality. See the [learn documentation](https://learn.microsoft.com/en-us/azure/ai-services/recover-purge-resources?tabs=azure-cli#purge-a-deleted-resource) for additional details.
+
 ```bash
 az group delete --name $RESOURCE_GROUP -y
 
 # Purge the soft delete resources
-az keyvault purge -n kv-${BASE_NAME}
-az openai purge -n oai-${BASE_NAME}  <-- TODO
-az aml purge -n amlw <-- TODO
-az ai services purge -n aih-${BASE_NAME} <--- TODO
-Role assignment removals
+SUBSCRIPTION_ID=$(az account show --query id --output tsv)
+az keyvault purge -n kv-${BASE_NAME} -l $LOCATION --no-wait
```

Comment:
Why is location needed now for key vault?  Was it always needed and the old command had a typo?

Reply:
The command reference says both name and location are optional which would I think is a documentation error.  (Name should be required at a minimum.)  I included both for completeness.  In practice name alone would probably be sufficient.

Reply:
yeah, key vault names are globally unique.  we use the previous command in lots of places without any issue.  I'd suggest just reverting as one less variable for someone to get wrong in this command :)